src/components/Alerts/src/alert.css

```diff
@@ -0,0 +1,151 @@
+}
+
+.denhaag-alert[variant="error"] .denhaag-button {
+  background-color: var(--denhaag-alert-error-icon-color);
+}
+
+.denhaag-alert[variant="error"] .denhaag-alert__close {
+  color: var(--denhaag-alert-error-close-icon-color);
+}
+
+.denhaag-alert[variant="info"] {
+  background-color: var(--denhaag-alert-info-background-color);
+}
+
+.denhaag-alert[variant="info"] .utrecht-heading-4 {
+  color: var(--denhaag-alert-info-title-color);
+}
+
+.denhaag-alert[variant="info"] .utrecht-paragraph {
+  color: var(--denhaag-alert-info-text-color);
+}
+
+.denhaag-alert[variant="info"] .denhaag-alert__icon {
+  color: var(--denhaag-alert-info-icon-color);
+}
+
+.denhaag-alert[variant="info"] .denhaag-button {
+  background-color: var(--denhaag-alert-info-action-button-color);
+}
+
+.denhaag-alert[variant="info"] .denhaag-alert__close {
+  color: var(--denhaag-alert-info-close-icon-color);
+}
+
+.denhaag-alert[variant="success"] {
+  background-color: var(--denhaag-alert-success-background-color);
+}
+
+.denhaag-alert[variant="success"] .utrecht-heading-4 {
+  color: var(--denhaag-alert-success-title-color);
+}
+
+.denhaag-alert[variant="success"] .utrecht-paragraph {
+  color: var(--denhaag-alert-success-text-color);
+}
+
+.denhaag-alert[variant="success"] .denhaag-alert__icon {
+  color: var(--denhaag-alert-success-icon-color);
+}
+
+.denhaag-alert[variant="success"] .denhaag-button {
+  background-color: var(--denhaag-alert-success-action-button-color);
+}
+
+.denhaag-alert[variant="success"] .denhaag-alert__close {
+  color: var(--denhaag-alert-success-close-icon-color);
+}
+
+.denhaag-alert[variant="warning"] {
+  background-color: var(--denhaag-alert-warning-background-color);
+}
+
+.denhaag-alert[variant="warning"] .utrecht-heading-4 {
+  color: var(--denhaag-alert-warning-title-color);
+}
+
+.denhaag-alert[variant="warning"] .utrecht-paragraph {
+  color: var(--denhaag-alert-warning-text-color);
+}
+
+.denhaag-alert[variant="warning"] .denhaag-alert__icon {
+  color: var(--denhaag-alert-warning-icon-color);
+}
+
+.denhaag-alert[variant="warning"] .denhaag-button {
+  background-color: var(--denhaag-alert-warning-action-button-color);
+}
+
+.denhaag-alert[variant="warning"] .denhaag-alert__close {
+  color: var(--denhaag-alert-warning-close-icon-color);
+}
+
+.denhaag-alert__content {
+  margin-inline-start: var(--denhaag-alert-content-margin-inline-start);
+  margin-inline-end: var(--denhaag-alert-content-margin-inline-end);
+}
+
+/* The position of the button is configurable with padding of alert__content and alert */
+.denhaag-alert__close button {
+  padding-inline-start: 0;
+  padding-inline-end: 0;
+  padding-block-start: 0;
+  padding-block-end: 0;
+}
+
+/* The position of the button is configurable with padding of alert__content and alert */
+.denhaag-alert__close button:focus {
+  outline-color: var(--denhaag-alert-action-focus-border-color);
+  outline-width: var(--denhaag-alert-action-focus-border-color);
```

Comment:
outline-width doesn't take a color as value. I think you forgot to change this one :D

Reply:
Whoops

Reply:
I also noticed just now that these tokens are not for action, but for close (button), so i've renamed them.